### ISSUE TITLE
[Bug] Link Utilities: Error on adding HowLongToBeat link

### ISSUE THREAD

**Issue by nikolay-borzov** [HerrKnarz/Playnite-Extensions#113]
**Describe the bug**
An error occurs when on "Add to link... > "HowLongToBeat":

```
18-08 23:59:36.480|ERROR|LinkUtilities#Log:Error loading data from HowLongToBeat|C:\Alex\Entwicklung\dotnet\Playnite-Extensions\Generic\LinkUtilities\Linker\LinkSources\LinkHowLongToBeat.cs|45
Newtonsoft.Json.JsonReaderException: Unexpected character encountered while parsing value: <. Path '', line 0, position 0.
   at Newtonsoft.Json.JsonTextReader.ParseValue()
   at Newtonsoft.Json.JsonTextReader.Read()
   at Newtonsoft.Json.JsonReader.ReadForType(JsonContract contract, Boolean hasConverter)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.Deserialize(JsonReader reader, Type objectType, Boolean checkAdditionalContent)
   at Newtonsoft.Json.JsonSerializer.DeserializeInternal(JsonReader reader, Type objectType)
   at Newtonsoft.Json.JsonConvert.DeserializeObject(String value, Type type, JsonSerializerSettings settings)
   at Newtonsoft.Json.JsonConvert.DeserializeObject[T](String value, JsonSerializerSettings settings)
   at LinkUtilities.Linker.LinkHowLongToBeat.GetSearchResults(String searchTerm) in C:\Alex\Entwicklung\dotnet\Playnite-Extensions\Generic\LinkUtilities\Linker\LinkSources\LinkHowLongToBeat.cs:line 45
```

**To Reproduce**
1. Find a game that has HowLongToBeat page, but don't have a link in Playnite
2. Open context menu on the game in the list > Link Utilities > Add to link... > HowLongToBeat
3. Open `extensions.log` in Playnite directory and examine last record

**Expected behavior**
Expect a link to HowLongToBeat added if the game has corresponding page. For example, there is page for [SuperTotalCarnage!](https://howlongtobeat.com/game/115048) (available for free at https://supertotalcarnage.indiegala.com/)

**please complete the following information:**
 - Windows version: `Windows 10 Pro`
 - Playnite version: `10.34`
 - Extension version: `4.0`

**Additional context**
- "Search link to..." also gives empty result.
- HowLongToBeat grabs data for the same game w/o problems

**Comment by HerrKnarz** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Repair HLTB link #113](https://trello.com/c/Ca2PNuSr/11-repair-hltb-link-113)

**Comment by HerrKnarz** [comment]
I finally found the time to look into the issue and decided to remove HLTB from the list of supported sites for the time being. The reason is, that Link Utilities already hijacked an API that's not supposed to be publicly available and HLTB changed something to make it even more difficult to use it from the outside. The HLTB addon itself found a rather hacky way to circumvent this but operates in a very gray area doing so.

Since it's also not possible to simply scrape the search results from the site itself and the HLTB-addon already provides a way to access the page of each game, I decided it's not worth the effort, if they don't want their data to be available outside of the website in the first place. I'll get back to it, once they hopefully release an official public API.
